src/Microsoft.AspNetCore.SignalR.Common/Protocol/StreamPlaceholder.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.AspNetCore.SignalR.Protocol
+{
+    public class StreamPlaceholder
```

Comment:
Not a fan of the name `StreamPlaceholder`

Reply:
Maybe `StreamIdWrapper`?

Reply:
StreamPlaceHolder is fine for now.